<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Titillium Web</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размер массива</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Array_size_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Sort_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Heap_sort_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Кучей</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Quick_sort_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Быстрой</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Merge_sort_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Слиянием</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Create_new_columns_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обновить массив</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bin Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите число</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bin_search_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Bin_search_number_input_line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Time_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>504</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>мс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>494</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прошло</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Index_output_line">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Индекс</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
